Geometry/GEMGeometryBuilder/python/gemGeometry_cff.py

```diff
@@ -2,4 +2,4 @@
 
 from Configuration.ProcessModifiers.dd4hep_cff import dd4hep
 
-dd4hep.toModify(gemGeometry, fromDDD = False, fromDD4hep = True)
+dd4hep.toModify(gemGeometry, useDDD = False, useDD4hep = True)
```

Comment:
I'm not sure why `fromDD4hep` is changed to `useDD4hep`. `fromDD4hep` has become a standard name. It's nice to be able to search CMSSW for `fromDD4hep` to see where DD4hep is enabled and where it is not enabled yet.

Reply:
The code was using useDD4hep and useDDD. The cff was wrong and it needs to be corrected.

Reply:
@cvuosalo Can you approve this PR. If you insist I shall change the names when I prepare the code for ME0.

Reply:
I was just wondering about it. The name can be left as-is. I don't think we have made a rule to only use the name `fromDD4hep`, though most migraters have used this name.